### ISSUE TITLE
Workspace switcher isn't keyboard accessible

### ISSUE THREAD

**Issue by Khaan25** [unkeyed/unkey#2507]
### Preliminary Checks

- [X] I have reviewed the documentation: https://unkey.com/docs

- [X] I have searched for existing issues: https://github.com/unkeyed/unkey/issues

- [X] This issue is not a question, general help request, or anything other than a bug report directly related to Unkey. Please ask questions in our Discord community: https://unkey.com/discord.

### Issue Summary

I was navigating the app.unkey.com via Keyboard.

I found that it's not accessible and I can't change workspace via keyboard.
There's no active, focused state on the buttons.

Only way to do is to do via Mouse.

### Steps to Reproduce

1. Go to app.unkey.com
2. Create a new workspace via team-switcher on top left in sidebar.
3. Try to navigate via keyboard

### Expected behavior

A User should be able to press `Tab` and select the workspace switcher and navigate through it via Keyboard. It should be accessible.

### Screenshots

![image](https://github.com/user-attachments/assets/6bc0f3fa-63a8-4bee-ac9f-c26ecc54ce66)

### Version info

```markdown
- OS: Windows
- Browser: Chrome
```

**Comment by chronark** [comment]
I wonder if shadcn can give us that for free

**Comment by Khaan25** [comment]
Yes, there's an example in shadcn for switcher or if you want I can build you a custom one with help of shadcn :)

**Comment by chronark** [comment]
Maybe once we have redesigned it in november

**Comment by chronark** [comment]
it's not really a bug, but a feature request I think
